### ISSUE TITLE
Microsoft Edge

### ISSUE THREAD

**Issue by HannahDelaney** [MacKenzieRose/ecommerce-pattern-library#9]
Everything is super clean, responsive, I like it!

One thing I noticed when clicking through the skip-links — you've got `tabindex=0` on some of your `li` elements and it's making them highlight twice. Small thing, but will help with accessibility to remove that. The skip-link font is also black on black for some reason, which is hard to read.

Looks great!

**Comment by MacKenzieRose** [comment]
100% agree; my tabbing and accessibility are so scary. Definitely something to improve on.

Thanks for testing :)
